pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -63,10 +64,11 @@ type Predictor struct {
 	inferenceServiceConfig *v1beta1.InferenceServicesConfig
 	deploymentMode         constants.DeploymentModeType
 	Log                    logr.Logger
+	forceStopRuntime       bool
 }
 
 func NewPredictor(client client.Client, clientset kubernetes.Interface, scheme *runtime.Scheme,
-	inferenceServiceConfig *v1beta1.InferenceServicesConfig, deploymentMode constants.DeploymentModeType,
+	inferenceServiceConfig *v1beta1.InferenceServicesConfig, deploymentMode constants.DeploymentModeType, forceStopRuntime bool,
```

Comment:
Since the `Reconcile` func receives an instance of the ISVC, for cleanliness and to avoid a new argument here, I wonder if we could resolve this flag from the provided ISVC on the `Reconcile` func?